api/src/main/java/org/openmrs/module/insuranceclaims/api/service/fhir/util/PractitionerUtil.java

```diff
@@ -0,0 +1,32 @@
+package org.openmrs.module.insuranceclaims.api.service.fhir.util;
+
+import org.hl7.fhir.dstu3.model.Reference;
+import org.openmrs.Provider;
+import org.openmrs.ProviderAttribute;
+import org.openmrs.module.fhir.api.util.FHIRConstants;
+import org.openmrs.module.fhir.api.util.FHIRUtils;
+import org.openmrs.module.insuranceclaims.api.model.InsuranceClaim;
+
+import static org.openmrs.module.insuranceclaims.api.service.fhir.util.InsuranceClaimConstants.PROVIDER_EXTERNAL_ID_ATTRIBUTE_UUID;
+
+public final class PractitionerUtil {
```

Comment:
do you need to do it as static method ? those utils above looks similar. I think it can be done with object programming